apps/participants/views.py

```diff
@@ -13,6 +13,8 @@
 )
 from rest_framework.throttling import UserRateThrottle
 from rest_framework_simplejwt.authentication import JWTAuthentication
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
```

Comment:
@afif-fahim can you reorder imports. We arrange imports lexicographically

Reply:
![image](https://user-images.githubusercontent.com/33936462/109049400-761a3100-7702-11eb-91a2-19a2228ce472.png)
Is this okay now? I am confused sorry! I followed `challenges/views.py`, `jobs/views.py` to arrange the imports.

Reply:
these two imports should go on line 3-4